mmv1/products/vertexai/terraform.yaml

```diff
@@ -97,6 +97,10 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         ignore_read: true
       name: !ruby/object:Overrides::Terraform::PropertyOverride
         custom_flatten: templates/terraform/custom_flatten/name_from_self_link.erb
+      monitoringConfig.snapshotAnalysis.stalenessDays: !ruby/object:Overrides::Terraform::PropertyOverride
+        default_from_api: true
+      monitoringConfig.snapshotAnalysis.monitoringInterval: !ruby/object:Overrides::Terraform::PropertyOverride
+        default_from_api: true
```

Comment:
[MEMO] I had a similar discussion on the default value in https://github.com/GoogleCloudPlatform/magic-modules/pull/6565#pullrequestreview-1131868943, but the reason why I added `default_from_api: true` to  `monitoringConfig.snapshotAnalysis.monitoringInterval` is that the google-beta provider test will fail because of the drift like below If we don't calculate the default value from the API. The field is a beta-only field, which plans to be deprecated. Please let me know if there is an alternative way to deal with the default value of the beta-only deprecated field.

```
  | After applying this test step, the plan was not empty.
  | stdout:
  | 
  | 
  | Terraform used the selected providers to generate the following execution
  | plan. Resource actions are indicated with the following symbols:
  |   ~ update in-place
  | 
  | Terraform will perform the following actions:
  | 
  |   # google_vertex_ai_featurestore_entitytype.entity will be updated in-place
  |   ~ resource "google_vertex_ai_featurestore_entitytype" "entity" {
  |         id           = "projects/kouzoh-p-kohama/locations/us-central1/featurestores/terraform2eshgfjs2g/entityTypes/terraform2eshgfjs2g"
  |         name         = "terraform2eshgfjs2g"
  |         # (4 unchanged attributes hidden)
  | 
  |       ~ monitoring_config {
  | 
  | 
  | 
  |           ~ snapshot_analysis {
  |               - monitoring_interval      = "86400s" -> null
  |                 # (3 unchanged attributes hidden)
  |             }
  |             # (3 unchanged blocks hidden)
  |         }
  |     }
  | 
  | Plan: 0 to add, 1 to change, 0 to destroy.
```

Reply:
`default_from_api` should be fine here. thanks!